tests/app/dao/test_fact_notification_status_dao.py

```diff
@@ -207,6 +207,7 @@ def test_fetch_notification_status_for_service_by_month(
     assert results[3].count == 1
 
 
+@pytest.mark.serial
 def test_fetch_notification_status_for_service_for_day(sample_service, sample_template, sample_notification):
```

Comment:
I think we usually add a comment about why a test is marked serial, but I'm not going to demand one be added.